WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -181,6 +189,7 @@ class WebKitViewController: UIViewController {
 
     private func setupRefreshButton() {
         let refreshButton = UIBarButtonItem(image: Gridicon.iconOfType(.refresh), style: .plain, target: self, action: #selector(WebKitViewController.refresh))
+        refreshButton.accessibilityLabel = NSLocalizedString("Refresh Webpage", comment: "Button to refresh the current webpage.")
```

Comment:
It's probably best to use “Reload” here so that it matches the Reload button's label in the Safari iOS app. What do you think?